CODEOWNERS

```diff
@@ -0,0 +1,56 @@
+# See also:
+# - https://github.com/blog/2392-introducing-code-owners
+# - https://help.github.com/articles/about-codeowners/
+
+# Each line is a file pattern followed by one or more owners (sorted alphabetically).
+# Please feel free to add yourself to a module or create a new mapping.
+# Ideally each module should have two or more owners.
+
+# Code owners are automatically requested for review
+# when someone opens a pull request that modifies code that they own.
+# Later matches take precedence.
+
+src/checkedint.d @redstar @andralex @JackStouffer
+
+src/core/atomic.d @WalterBright @aG0aep6G
+src/core/attribute.d @JacobCarlborg
+src/core/bitop.d @schveiguy @tsbockman @mathias-lang-sociomantic
+src/core/cpuid.d @WalterBright @MarcoLeisse @JackStouffer
+src/core/demangle.d @WalterBright @MartinNowak @rainers @ibuclaw
+src/core/exception.d @MartinNowak @WalterBright @jmdavis @CyberShadow
+src/core/internal @MartinNowak @schveiguy
+src/core/math.d @braddr @redstar
+src/core/runtime.d @MartinNowak @Abscissa
+src/core/simd.d @WalterBright @MartinNowak
+src/core/stdc/* @schveiguy
+src/core/stdcpp/* @WalterBright @Darredevil
+src/core/sync/* @MartinNowak @mathias-lang-sociomantic @WalterBright
+src/core/sys/bionic/* @joakim-noah
+src/core/sys/darwin/* @klickverbot
+src/core/sys/freebsd/* @redstar @MartinNowak @Calrama @jmdavis
+src/core/sys/linux/* @Burgos @redstar @MartinNowak
+src/core/sys/netbsd/* @nrTQgc @joakim-noah
+src/core/sys/openbsd/* @redstar
+src/core/sys/osx/* @etcimon @MartinNowak
+src/core/sys/posix/* @CyberShadow @MartinNowak @joakim-noah @redstar
+src/core/sys/solaris/* @redstar
+src/core/sys/windows/* @CyberShadow
+src/core/thread.d @MartinNowak @ZombineDev @Burgos @jpf91
```

Comment:
Can you move me at the end of the list? There much more knowledgeable people than me for this part of the codebase.